src/templates/content.html

```diff
@@ -40,6 +40,9 @@ <h1>Video Index</h1>
         <p>Video ID: {{ video.video_id }}</p>
         <p>Video Extension: {{ video.extension }}</p>
         <p>User ID: {{ video.user_id }}</p>
+        <a href="/delete/{{ video.video_id }}.{{ video.extension }}">
+          <button class="logout-button" type="button">Delete</button>
```

Comment:
Change class to `delete-button` or something like that. It's not actually being used for styling, so changing the class will better reflect the purpose without any consequences.

Reply:
Changed!